includes/api/ApiQueryLogEvents.php

```diff
@@ -169,8 +169,13 @@ public function execute() {
 			}
 
 			if ( $this->fld_user ) {
-				$row->user_name = isset($userNames[$row->log_user]) ?
-					$userNames[$row->log_user] : 'unknown';
+				if ( isset( $userNames[$row->log_user] ) ) {
+					$row->user_name = $userNames[$row->log_user];
+				} else {
+					$row->user_name = 'unknown';
+					\Wikia\Logger\WikiaLogger::instance()
+						->warning( "User with id {$$row->log_user} was not found" );
```

Comment:
Why `$$`? :)

Reply:
I'd rather use a fixed log message, e.g. `User not found by a given ID` and log the ID in the log message context.

Reply:
`$$` was a typo, fixed in https://github.com/Wikia/app/pull/13912/commits/52327bbc86529d1d76ca2b0b5f9fc5e9e26874fc